bedrock/firefox/tests/test_base.py

```diff
@@ -1121,22 +1121,17 @@ def setUp(self):
                       'Gecko/20100101 Firefox/35.0')
         self.url = reverse('firefox.hello.start', args=['35.0'])
 
-    def test_fx_hello_no_conversation(self):
-        """Should identify when there is no conversation"""
+    def test_fx_hello_redirect_non_firefox(self):
+        """Should redirect to /firefox/new if not on Firefox"""
```

Comment:
I believe you're right, but this can't hurt. If this redirect is important for this page it would be good to know if someday it changes for some reason.